charts/argocd-apps/values.yaml

```diff
@@ -9,6 +9,6 @@ globalHelmValues:
   govukStack: blue  # TODO: eliminate the need for govukStack.
   externalDomainSuffix: ""
   # TODO: delete internalDomainSuffix once Plek handles empty GOVUK_APP_DOMAIN.
-  internalDomainSuffix: apps.svc.cluster.local.
+  internalDomainSuffix: apps.svc.cluster.local
```

Comment:
Removed trailing `.`

Reply:
The trailing dot serves a useful purpose here - it prevents unnecessary lookups. See e.g. https://pracucci.com/kubernetes-dns-resolution-ndots-options-and-why-it-may-affect-application-performances.html

Ultimately I'm hoping we can remove these suffixes entirely but I need to fix Plek first.

Reply:
Was looking at some of the logs for apps and they where generating paths using internal suffix with trailing . such as content-store.apps.svc.cluster.local./somepath.
It seemed it was causing problem in finding the path - but will need to do more testing - i can re-add the trailing dot for now and test locally with and without it?

Reply:
Oh, if that's appearing in markup (like links in HTML and stuff like that) then that sounds like we might be using the internal suffix someplace where we should be using the external one 🤔

If it's breaking stuff then of course feel free to drop the dot and we'll figure out what's going on later.